docs/managed_package.rst

```diff
@@ -0,0 +1,247 @@
+
+
+
+Test a Beta Version
+-------------------
+
+The ``ci_beta`` flow installs the latest beta version of the project on a scratch org, and runs Apex tests against it.
+
+.. code-block:: console
+
+    $ cci flow run ci_beta --org packaging 
+
+This flow is intended to be run whenever a beta release is created.
+
+
+
+Generate Release Notes
+----------------------
+
+The ``github_release_notes`` task fetches the text from pull requests that were merged between two given tags. The task then searches for specific titles (Critical Changes, Changes, Issues Closed, New Metadata, Installation Info, and so on) in the pull request bodies, and aggregates the text together under those titles in the GitHub tag description.
+
+To see what the release notes look like without publishing them to GitHub:
+
+.. code-block::
+
+    $ cci task run github_release_notes --tag release/1.2
+
+.. note:: The ``--tag`` option indicates which release's change notes are aggregated. The previous command aggregates all change notes between the `1.2` release and the `1.1` release.
+
+To see where each line in the release notes comes from, use the ``--link_pr True`` option.
+
+.. code-block::
+
+    $ cci task run github_release_notes --tag release/1.2 --link_pr True
+
+To publish the release notes to a release tag in GitHub, use the ``--publish True`` option:
+
+.. code-block::
+
+    $ cci task run github_release_notes --tag release/1.2 --publish True
+
+To use additional headings, add new ones (as parsers) under the ``project__git__release_notes`` section of the ``cumulusci.yml`` file.
+
+.. code-block::
+
+    release_notes:
+        parsers:
+            7: class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+
+.. note:: The new parser is listed with the number ``7`` because the first six are the `default parsers <https://github.com/SFDO-Tooling/CumulusCI/blob/671a0e88cef79e9aeefe1e2b835816cd8141bdbb/cumulusci/cumulusci.yml#L1154>`_ that come with CumulusCI.
+        
+
+
+Upload and Test a Final Version
+-------------------------------
+
+To upload a production release of your managed package project:
+
+.. code-block::
+
+    $ cci flow run release_production --org packaging 
+
+Similar to ``release_beta``, this task uploads a new production version of your package, creates a release tag in GitHub, and aggregates release notes for the new version.
+
+.. important::
+
+    This flow assumes that the package contents have previously been deployed using the ``ci_master`` flow.
+
+To upload the new production version without creating the GitHub tag and generating release notes:
+
+.. code-block::
+
+    $ cci task run upload_beta --name v1.2.1 --production True
+
+To test the new package version:
+
+.. code-block::
+
+    $ cci flow run ci_release
+
+The ``ci_release`` flow installs the latest production release version, and runs the Apex tests from the managed package on a scratch org.
+
+
+
+Publish an Install Plan to MetaDeploy
```

Comment:
This section doesn't provide, and we haven't provided anywhere else, details of how to create a MetaDeploy plan. I'm afraid it may add more confusion than it helps until we can share more content.

Reply:
I'm ok with removing this.